doc/rados/operations/monitoring.rst

```diff
@@ -738,3 +738,39 @@ Print active connections and their TCP round trip time and retransmission counte
 
 	248     89      1       mgr.0   863     1677    0
 	3       86      2       mon.0   230     278     0
+
+Tracking Data Availability Score of a Cluster
```

Comment:
I found this file to be the best place to add docs for this feature as per my best judgement. Please let me know if these should be moved someplace else.

Reply:
I also wasn't sure how to highlight that this feature is in tech preview, any help here would be great! Do we use a special kind of banner or just mentioning it in text is fine?

Reply:
The location for the file is good.

Reply:
Added that this feature is in tech preview in the release notes.